build_variables.bzl

```diff
@@ -949,6 +949,8 @@ libtorch_python_core_sources = [
     "torch/csrc/utils/object_ptr.cpp",
     "torch/csrc/utils/python_arg_parser.cpp",
     "torch/csrc/utils/python_dispatch.cpp",
+    "torch/csrc/utils/pytree.cpp",
+    "torch/csrc/utils/pytree_bindings.cpp",
```

Comment:
@wconstab Since we're not using cpp pytrees in the branch anymore, should we just revert these changes entirely?